docs/development.md

```diff
@@ -69,7 +58,68 @@ of FCOS.
 After a change is landed in FCOS, it's often OK to also immediately update RHCOS' main
 branch to the latest FCOS.  We try to keep things compatible.
 
-```
+```bash
 $ (cd fedora-coreos-config && git fetch origin && git reset --hard origin/testing-devel)
 $ git commit -a -m 'Update FCOS'
 ```
+
+## Debugging failures in app.ci
+
+Occasionally it is necesasry to debug test failures on the [OpenShift CI clusters](https://docs.ci.openshift.org/docs/getting-started/useful-links/#clusters)
+by scheduling a `cosa` pod on the cluster.  There's a few additional steps after
+getting the pod scheduled, in order to be able to build and test RHCOS.
+
+1. Create the pod
+
+Using the YAML below, create a new pod with `oc create -f pod.yaml`
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  # Replace this with e.g. $USER-cosa
+  name: test-cosa
+spec:
+  containers:
+  - args:
+    - shell
+    - sleep
+    - infinity
+    image: quay.io/coreos-assembler/coreos-assembler:latest
```

Comment:
Heh, the funny thing is that now this is actually built in app.ci itself. This is fine, but it might be better to use that FQIN instead. That reduces the chance of using a different image than what the tests are since Quay.io is delayed.